MoneyTransferClient/RunWorkflow.cs

```diff
@@ -0,0 +1,35 @@
+using MoneyTransferProject;
+using Temporalio.Client;
+
+// Connect to the Temporal server
+var client = await TemporalClient.ConnectAsync(new("localhost:7233"));
+
+// Define payment details
+var details = new PaymentDetails(
+    "85-150", // SourceAccount
+    "43-812", // TargetAccount
+    400,      // Amount
+    "12345"   // ReferenceID
+);
+
+Console.WriteLine($"Starting transfer from account {details.SourceAccount} to account {details.TargetAccount} for ${details.Amount}");
+
+var workflowId = $"pay-invoice-{Guid.NewGuid()}";
+
+try
+{
+    // Start the workflow
+    var handle = await client.StartWorkflowAsync(
+        (MoneyTransferWorkflow wf) => wf.RunAsync(details),
+        new(id: workflowId, taskQueue: "MONEY_TRANSFER_TASK_QUEUE"));
+
+    Console.WriteLine($"Started Workflow {workflowId}");
+
+    // Await the result of the workflow
+    var result = await handle.GetResultAsync<string>();

```

Comment:
```suggestion
    var result = await handle.GetResultAsync();
```

The type should be able to be inferred